src/utils/compare-events/index.ts

```diff
@@ -0,0 +1,29 @@
+interface PartialEvent {
+  startDate: Date
+  endDate: Date
+}
+
+/**
+ * Compares 2 events. Returns negative if a comes first, positive if b comes first
+ * @param a First Event
+ * @param b Second Event
+ */
+export const compareEvents = (a: PartialEvent, b: PartialEvent) => {
```

Comment:
Can we make it so it sorts it like this:

If a.week and b.week are the same and not `undefined`, sort them alphabetically

Otherwise, use the logic you already have

I could add that in a follow up PR if you want